.changeset/lazy-elephants-refuse.md

```diff
@@ -0,0 +1,12 @@
+---
+"@wso2is/admin.tenants.v1": minor
+"@wso2is/admin.administration.v1": minor
```

Comment:
there's no changes to this administrators package in this pr, hence we can remove it I guess.

Reply:
I generate this using pnpm changeset.. I'll recheck

Reply:
Sometimes changesets CLI adds incorrect packages when the feature branch is not synced with master branch. This may be due to that.